supreme_court_predictions/statistics/descriptives.py

```diff
@@ -64,7 +65,8 @@ def fix_indices(self, main, sub):
             new_indices.append((main, index))
         return pd.MultiIndex.from_tuples(new_indices)
 
-    def get_count_desc(self, df, cols):
+    @staticmethod
+    def get_count_desc(df, cols):
```

Comment:
Made method static since it does not reference the `self` variable at all. 

What is a static method: [Link](https://www.digitalocean.com/community/tutorials/python-static-method)